api/migrations/20200618184040-keywordIndexUpdate.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+let dbm;
+let type;
+let seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  let mClient;
+  try {
+    mClient = await db.connection.connect(db.connectionString, { native_parser: true });
+    const nrptiCollection = mClient.collection('nrpti')
+
+    nrptiCollection.dropIndex('keyword-search-text-index');
+
+    nrptiCollection.createIndex(
+    {
+      summary: 'text',
+      location: 'text',
+      'issuedTo.companyName': 'text',
+      'issuedTo.firstName': 'text',
+      'issuedTo.middleName': 'text',
+      'issuedTo.lastName': 'text',
+      'issuedTo.fullName': 'text',
+      permitNumbers: 'text',
+      recordName: 'text',
+      name: 'text'
```

Comment:
Are these all the fields we want to index? The ticket doesn't seem to say what we should be indexing on. Would it be a good idea to add description, permittee, type, status and region? That way we can search on the columns that we see on the mines list.

Reply:
The ticket only indicates Name and Permit Numbers. It's easy enough to add other fields if requested of course.